src/structures/FormTextInput.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const BaseMessageComponent = require('./BaseMessageComponent');
+const { FormInputTextStyles } = require('../util/Constants');
+const Util = require('../util/Util');
+
+/**
+ * @typedef {'SHORT'|'PARAGRAPH'} FormInputTextStyle
+ */
+
+/**
+ * Represents a text input component.
+ * @extends {BaseMessageComponent}
+ */
+class FormTextInput extends BaseMessageComponent {
+  constructor(data = {}) {
+    super({ type: 'INPUT_TEXT' });
+
+    /**
+     * The style of this text input
+     * @type {FormInputTextStyle}
+     */
+    this.style = data.style ? FormTextInput.resolveStyle(data.style) : null;
+
+    /**
+     * The customId for this text input
```

Comment:
To match other custom id descriptions
```suggestion
     * A unique string to be sent in the interaction when submitted
```